SetupDataPkg/ConfApp/SetupConf.c

```diff
@@ -168,6 +168,122 @@ PrintOptions (
   return Status;
 }
 
+/**
+  Set arbitrary SVD values to variable storage
+
+  @param Value     a pointer to the variable list
+  @param ValueSize Size of the data for this setting.

```

Comment:
nit: param are in different order in header than in function

Reply:
Fixed.